docs/baking.md

```diff
@@ -0,0 +1,159 @@
+<!--
+   - SPDX-FileCopyrightText: 2019 TQ Tezos <https://tqtezos.com/>
+   -
+   - SPDX-License-Identifier: LicenseRef-MIT-TQ
+   -->
+# Baking with tezos-packaging on Ubuntu
+
+Tezos-packaging provides an easy way to install and set up infrastructure for
+interacting with Tezos blockchain. This article provides a step-by-step guide for
+setting up baking instance for Tezos on Ubuntu.
+
+## Prerequisites
+
+### Installing required packages
+
+In order to run baking instance, you'll need the following Tezos binaries:
```

Comment:
```suggestion
In order to run a baking instance, you'll need the following Tezos binaries:
```

Reply:
Articles have always been tricky for me :joy: :

Reply:
Naaa, these are just minor fixes, overall it's well written IMO